sktime/annotation/adapters/_sktime_od.py

```diff
@@ -0,0 +1,110 @@
+# -*- coding: utf-8 -*-
+from sktime.annotation.base._base import BaseSeriesAnnotator
+import pandas as pd
+import numpy as np
+from sklearn import clone
+from sktime.forecasting.base import ForecastingHorizon
+
+__author__ = ["satya-pattnaik", "mloning"]
+
+
+class SktimeODAnnotator(BaseSeriesAnnotator):
```

Comment:
We need to come up with a better name I think, how about simply `OutlierDetector`? I don't have a good idea on how to capture the fact that a forecaster is used internally. Any other suggestions?

Reply:
Any suggestions on a better name @fkiraly ?